transports/webrtc-websys/CHANGELOG.md

```diff
@@ -1,7 +1,7 @@
-## 0.4.0-alpha.2
+## 0.4.0
```

Comment:
Should we also update the changelog to reflect the removal of alpha status?

Reply:
yeah but this was already done in 23988477e0d67e72d86ba21895a102d93aecfd8c, don't understand how you were able to comment aha